pkg/gardener/secret/client.go

```diff
@@ -0,0 +1,67 @@
+package secret
```

Comment:
Why the package name is `secret`?

Reply:
Because it deals with k8s `secrets`. It is similar to `node`, `svc` and `shoot`. I can make plural though. Will it make sense?